src/main/java/frc/robot/subsystems/vision/VisionIO.java

```diff
@@ -22,4 +24,7 @@ public static class VisionIOInputs {
   }
 
   public default void updateInputs(VisionIOInputs inputs) {}
+
+  public default void setSimPose(
```

Comment:
is there a way not to have sim specific functionality in the io interface? if not isok but avoiding it is good 👍